bundles/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/internal/influx1/InfluxDB1RepositoryImpl.java

```diff
@@ -131,7 +130,7 @@ public boolean write(List<InfluxPoint> influxPoints) {
             BatchPoints batchPoints = BatchPoints.database(configuration.getDatabaseName())
                     .retentionPolicy(configuration.getRetentionPolicy()).points(points).build();
             currentClient.write(batchPoints);
-        } catch (InfluxException | InfluxDBIOException e) {
+        } catch (InfluxException | InfluxDBException e) {
```

Comment:
Should this also be changed for method `query` in line 179?

Reply:
At least it can't hurt.